### PR TITLE
Separate Loadout Functions From Respawn Functions in Custom Gamemodes

### PR DESCRIPTION
**Pull request by DBmaoha**
Required by https://github.com/R2Northstar/NorthstarMods/pull/567
Separent custom loadout functions from respawn fucntion, so fixed grace period won't mess up their loadout

### CODE REVIEW

File: Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_inf.gnut
Comment (F1F7Y) [inline Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_inf.gnut:141]: whitespace change

**Review by F1F7Y — CHANGES_REQUESTED** [review]
Works in testing, code looks good.

Just 1 unnecessary line change, will approve when fixed

**Review by F1F7Y — APPROVED** [review]
Code looks good, tested `sns` before [ef0e222](https://github.com/R2Northstar/NorthstarMods/pull/630/commits/ef0e222dced4bdabcd4f7588e8b4aabcc7ef7a74) and all worked as it should

File: Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_chamber.nut
Comment (GeckoEidechse) [inline Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_chamber.nut:15]: @F1F7Y we probably don't want leftover commented out code in codebase, right?

Comment (F1F7Y) [reply Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_chamber.nut:15]: If it's documented with a comment id say it can stay. My thinking was that one day:tm: we'll format this so we don't really need to push for perfection right now

Comment (GeckoEidechse) [reply Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_chamber.nut:15]: I mean, it's not really _perfection_ but just proper cleanup before merge :P

**Review by ASpoonPlaysGames — CHANGES_REQUESTED** [review]

File: Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_inf.gnut
Comment (DBmaoha) [reply Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_inf.gnut:111]: yes, these are not loadout functions, stim and sound effect is bind with player's respawn and shouldn't be played again each time player gets a new loadout

Comment (ASpoonPlaysGames) [reply Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_inf.gnut:111]: > yes, these are not loadout functions, stim and sound effect is bind with player's respawn and shouldn't be played again each time player gets a new loadout

Does this account for the first infected player, as they aren't killed when they are infected?

Comment (ASpoonPlaysGames) [reply Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_inf.gnut:111]: Thanks for the clarification

**Pull request closed without merging**
